packages/@lwc/shared/package.json

```diff
@@ -1,6 +1,11 @@
 {
     "name": "@lwc/shared",
     "version": "1.5.2",
+    "author": "LWC Team",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/salesforce/lwc.git"
+    },
```

Comment:
This is a public-private package so I don't think we want to make it easier for crawlers to find it.